commerce-account-web/src/main/java/com/liferay/commerce/account/web/internal/frontend/CommerceAccountAddressClayTable.java

```diff
@@ -92,27 +92,25 @@
 
 		Address address = (Address)model;
 
-		StringBundler sb = new StringBundler(7);
+		StringBundler sb = new StringBundler(3);
 
-		sb.append("javascript:deleteCommerceAddress");
-		sb.append(StringPool.OPEN_PARENTHESIS);
-		sb.append(StringPool.APOSTROPHE);
+		sb.append("deleteCommerceAddress('");
 		sb.append(address.getAddressId());
-		sb.append(StringPool.APOSTROPHE);
-		sb.append(StringPool.CLOSE_PARENTHESIS);
-		sb.append(StringPool.SEMICOLON);
+		sb.append("');");
 
 		ClayTableAction deleteClayTableAction = new ClayTableAction(
-			StringPool.BLANK, sb.toString(), StringPool.BLANK,
-			LanguageUtil.get(httpServletRequest, "delete"), false, false);
+			StringPool.BLANK, StringPool.POUND, StringPool.BLANK,
+			LanguageUtil.get(httpServletRequest, "delete"), sb.toString(),
+			false, false);
 
 		clayTableActions.add(deleteClayTableAction);
 
-		sb.setStringAt("javascript:editCommerceAddress", 0);
+		sb.setStringAt("editCommerceAddress('", 0);
```

Comment:
since we're doing `setStringAt` on this, I would be ok simplifying that one string (and still appending the `StringPool` elements separately) to make this a little simpler / less error prone. But I'm ok either way.